.rhiza/tests/stress/test_git_stress.py

```
@@ -0,0 +1,169 @@
+
+@pytest.mark.stress
+def test_concurrent_git_status(root: Path, concurrent_workers: int):
+    """Test concurrent git status operations.
+
+    Verifies that multiple git status commands can run concurrently
+    without conflicts.
+    """
+
+    def run_git_status():
+        """Run git status and return success status."""
+        result = subprocess.run(  # nosec
```

Ruff (0.15.14)

[error] `subprocess` call: check for execution of untrusted input

(S603)